web/src/components/base/Link.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { helpers } from '../../constants/theme'
+import isExternalUrl from '../../utils/isExternalUrl'
+
+const StyledLink = styled(RouterLink)<{ $removeHighlight?: boolean; $adaptiveFontSize?: boolean }>`
+  color: inherit;
+  text-decoration: none;
+  display: flex;
+
+  ${props => props.$removeHighlight && helpers.removeLinkHighlighting}
+  ${props => props.$adaptiveFontSize && helpers.adaptiveFontSize}
+`
+
+const StyledAnchor = styled.a`
+  color: inherit;
+  text-decoration: none;
+`
+
+type LinkProps = {
+  to?: string
+  children: ReactNode
+  ariaLabel?: string
+  className?: string
+  newTab?: boolean
+  removeHighlight?: boolean
+  adaptiveFontSize?: boolean
+  href?: string
+  isCleanAnchor?: boolean
+}
+
+const Link = ({
+  to,
+  href,
+  children,
+  ariaLabel,
+  isCleanAnchor,
+  className,
+  newTab,
+  removeHighlight,
+  adaptiveFontSize,
+}: LinkProps): ReactElement => {
+  const newTabProps = { target: '_blank', rel: 'noopener noreferrer' }
+
+  const getUrl = () => {
+    if (to) {
+      return to
+    }
+    if (href) {
+      return href
+    }
+    return '/'
+  }
```

Comment:
:x: This allows for wrong usage of the component since neither `to` nor `href` are required. Personally I'd favor to unify the behavior and always just have either `to` or ` href` and remove the other prop completely.

Another option to make this typesafe (also less unified/more confusing) you could do something like:
``` ts
type ToProps = {
  href: string
  to: undefined
} | {
  href: undefined
  to: string
}
```

However, I think only having either `to` or `href`` would be the better choice since
- less logic needed
- streamlined usage
- no confusion that different properties have the same result

Personally I don't mind whether we keep `to` or `href`.